well-sensor-esp32.ino

```diff
@@ -83,7 +85,16 @@ void loop() {
   mqttClient.loop();
   
   dance(100);
+
+  if (current_sample < SIZE) {
+    
+    data[current_sample++] = getNumber();
+    return;
+  }
   
-  
-  
+  int res = filterData();
+  char payload[5];
+  sprintf(payload, "%d", res);
+  mqttClient.publish("data", payload);
+  current_sample = 0;
```

Comment:
Pls introduce function for this in init-data.h

Reply:
The same error as for previous comment. 
Cause of function dependency(from filter-median.h) this code can`t be moved to init-data.h

Reply:
same approach - single responsibility
1st function handles data geathering
2nd function handles mqtt